src/engraving/libmscore/layoutbreak.cpp

```diff
@@ -112,7 +112,7 @@ void LayoutBreak::read(XmlReader& e)
             readProperty(e, Pid::START_WITH_LONG_NAMES);
         } else if (tag == "startWithMeasureOne") {
             readProperty(e, Pid::START_WITH_MEASURE_ONE);
-        } else if (tag == "firstSystemIdentation") {
+        } else if (tag == "firstSystemIndentation" || tag == "firstSystemIdentation") {
```

Comment:
This is the API change -- I've split it into its own commit so it could be addressed separately.